### PR TITLE
Updated May-1 preview to include changes from March-1 preview

### PR DESCRIPTION
**Pull request by abhahn**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/examples/conversations/Orchestrator_direct_target.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Content-Type": "application/json",
+    "api-version": "2021-11-01-preview",
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/examples/conversations/Orchestrator_direct_target.json:6]: Should be 2022-05-01-preview

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json [hunk 2]
```diff
@@ -0,0 +1,1430 @@
+{
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json:1]: Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvYW5hbHl6ZWNvbnZlcnNhdGlvbnMuanNvbiIsInRhcmdldFVybCI6Imh0dHBzOi8vcmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbS9BenVyZS9henVyZS1yZXN0LWFwaS1zcGVjcy85OTIwOTlkZmJkNTJmZjVlYzcwMGFjZjg5ZDlmNzhkNzliNjE5OGNlL3NwZWNpZmljYXRpb24vY29nbml0aXZlc2VydmljZXMvZGF0YS1wbGFuZS9MYW5ndWFnZS9wcmV2aWV3LzIwMjItMDUtMDEtcHJldmlldy9hbmFseXplY29udmVyc2F0aW9ucy5qc29uIn0=

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json [hunk 3]
```diff
@@ -0,0 +1,248 @@
+{
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json:1]: Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvY29tbW9uLmpzb24iLCJ0YXJnZXRVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvOTkyMDk5ZGZiZDUyZmY1ZWM3MDBhY2Y4OWQ5Zjc4ZDc5YjYxOThjZS9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTA1LTAxLXByZXZpZXcvY29tbW9uLmpzb24ifQ==

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json [hunk 4]
```diff
@@ -0,0 +1,2903 @@
+{
```
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json:1]: Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvdGV4dGFuYWx5dGljcy5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzLzk5MjA5OWRmYmQ1MmZmNWVjNzAwYWNmODlkOWY3OGQ3OWI2MTk4Y2Uvc3BlY2lmaWNhdGlvbi9jb2duaXRpdmVzZXJ2aWNlcy9kYXRhLXBsYW5lL0xhbmd1YWdlL3ByZXZpZXcvMjAyMi0wNS0wMS1wcmV2aWV3L3RleHRhbmFseXRpY3MuanNvbiJ9

**Review by heaths — APPROVED** [review]
Apart from the one example with the wrong api-version, OpenAPI Hub's diff didn't show any semantic changes from 2022-03-01-preview here. Is that expected, though?

I'm still concerned that 2022-05-01 (stable) will ship with breaking changes for textanalytics.json that aren't in here. And, though we can do it in another PR, this preview really shouldn't be 2022-05-01-preview because that makes it seem like the preview before 2022-05-01 (stable) which goes against guidelines.

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/examples/conversations/Orchestrator_direct_target.json (hunk 1 above)
Comment (abhahn) [reply specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/examples/conversations/Orchestrator_direct_target.json]: Should be fixed now!

**Comment by abhahn** [comment]
> Apart from the one example with the wrong api-version, OpenAPI Hub's diff didn't show any semantic changes from 2022-03-01-preview here. Is that expected, though?
> 
> I'm still concerned that 2022-05-01 (stable) will ship with breaking changes for textanalytics.json that aren't in here. And, though we can do it in another PR, this preview really shouldn't be 2022-05-01-preview because that makes it seem like the preview before 2022-05-01 (stable) which goes against guidelines.

Technically yes, we should have a semantic diff between May-1 and March-1, but the changes which should be applied for this will be merged with April-1.  This PR will allow Bidisha to continue her work on the conversations spec without missing any changes, but we will need to bring the changes from April-1 over once the work for that release is finished.  I'll follow up today to see if we can get that merged ASAP.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@JeffreyRichter or @johanste can we get a breaking change exemption. This is the same as 2022-03-01-preview where the `error` property of the error object was made required. Honestly not sure why we're still seeing it, though. Perhaps after we can GA a newer version of common.json (should for 2022-05-01) it'll go away.

@jhendrixMSFT @lmazuel the Avacado error about a missing tag can also be ignored. I tried to suppress it, but haven't been successful. Know of any way? The other Avocado errors should be an easy fix, though. @abhahn just needs to remove the analyzeconveration.json file and examples/conversation subdirectory if not part of this api-version.

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json (hunk 2 above)
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json:1]: No changes were detected.

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json (hunk 4 above)
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/textanalytics.json:1]: No changes were detected.

File: specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json (hunk 3 above)
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json:1]: No changes were detected.

**Comment by heaths** [comment]
@abhahn alternatively, should analyzeconversations.json be in the readme.md since @bidisha-c is making changes to them against this same target branch? Is that true for April as well?

**Pull request merged**